parser/parser_test.go

```diff
@@ -57,434 +53,434 @@ func getFactTestCases() []testCase {
 	}
 }
 
-func getRuleTestCases() []testCase {
-	t1, _ := time.Parse(time.RFC3339, time.Now().Format(time.RFC3339))
-	t2, _ := time.Parse(time.RFC3339, time.Now().Add(2*time.Second).Format(time.RFC3339))
+// func getRuleTestCases() []testCase {
```

Comment:
I assume these tests need to be rewritten?

Reply:
Yes, the parser is totally broken right now, and there are still some ongoing changes to the syntax of the new expressions. I'll fix it when we'll have a more stable syntax.